_posts/2015-09-20-Continuous-Integration.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout:     post
+title:      "Continuous Integration"
+subtitle:   "Why CI is such a rocking thing?"
+date:       2015-09-20 14:00:00
+author:     "Faisal Farooq"
+header-img: "img/fsl_post.png"
+---
+
+## Background on Faisal
+Faisal is currently attending the Firehose Project coding bootcamp and learned about continuous integration during the capstone project.
+
+Say Hello to Faisal [here](http://www.twitter.com/ffarooq00).
+
+{% include youtubePlayer.html id="905fwucM-Kc" %}
+
+Watch the full video [here](https://www.youtube.com/watch?v=905fwucM-Kc).
```

Comment:
This should be a link to the full lightning talk. 

`{% include youtubePlayer.html id="rQ8iLTFZyz0" %}`